src/popup/pages/Popups/MessageSign.vue

```diff
@@ -1,29 +1,34 @@
 <template>
   <Modal
+    show
     full-screen
     class="message-sign"
     data-cy="popup-aex2"
   >
     <TransactionInfo
       :custom-title="$t('pages.popupMessageSign.title')"
-      :sender="{ name: app.name, address: app.host, url: app.url }"
+      :sender="{
+        name: popupProps?.app?.name,
+        address: popupProps?.app?.host,
+        url: popupProps?.app?.url
+      }"
```

Comment:
I suggest to move this into composable and set the correct type for the return value to enforce correct data structure.